python/sbp/client/forwarder.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Mark Fine <mark@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+from threading import Thread
+
+class Forwarder(Thread):
```

Comment:
It works for me.  To what resources are you referring? What is the use case for using the driver other than through its handler?  If it really makes sense to do this then you may need a mutex, but it'd make more sense to include the mutex in the driver implementation.